--- a/node_modules/@bittingz/expo-widgets/ios/ExpoWidgetsModule.swift
+++ b/node_modules/@bittingz/expo-widgets/ios/ExpoWidgetsModule.swift
@@ -15,7 +15,7 @@
         Name("ExpoWidgets")
         
         Function("setWidgetData") { (data: String) -> Void in   
-            let logger = Logger()
+            let logger = log
             // here we are using UserDefaults to send data to the widget
             // you MUST use a suite name of the format group.{your project bundle id}.expowidgets
             let widgetSuite = UserDefaults(suiteName: "group.expo.modules.widgets.example.expowidgets")
